crypt_test.go

```diff
@@ -239,49 +239,58 @@ func (ctb *cryptTestBase) checkRecoveryActivationData(c *C, prefix, path string,
 	c.Check(err, Equals, ErrNoActivationData)
 }
 
-type cryptTPMTestBase struct {
+type cryptTPMSimulatorSuite struct {
+	testutil.TPMSimulatorTestBase
 	cryptTestBase
 
 	keyFile        string
 	authPrivateKey TPMPolicyAuthKey
 }
 
-func (ctb *cryptTPMTestBase) setUpTestBase(c *C, ttb *testutil.TPMTestBase) {
-	ctb.cryptTestBase.setUpTestBase(c, &ttb.BaseTest)
+var _ = Suite(&cryptTPMSimulatorSuite{})
+
+func (s *cryptTPMSimulatorSuite) SetUpSuite(c *C) {
```

Comment:
can't these noisy changes be done separately and similar other ones? It's generally a bad idea to mix lots of small cosmetic changes and important changes together.